src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -105,12 +108,98 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
+            sx={{
+              // Matches the styles usually applied in DashboardContentContainer.
+              // (Moved in here because of the Palette drawer)
+              maxWidth: `${theme.breakpoints.values.xl}px`,
+              pt: 2,
+              pb: 8,
+              px: { xs: 1, sm: 3, lg: 4 },
+            }}
+          >
+            <FormBuilderHeader
+              formDefinition={formDefinition}
+              lastUpdatedDate={lastUpdatedDate}
+              onClickPreview={onClickPreview}
+            />
+            <Box sx={{ p: 4 }}>
+              <FormTree
+                definition={workingDefinition}
+                onEditClick={(item: FormItem) => {
+                  function editItem() {
+                    setSelectedItem(item);
+                  }
+
+                  if (dirty) {
+                    // React's useState accepts either a value or a function that yields a value.
+                    // In this case, we want the function itself to *be* the state value, which is the reason
+                    // for defining `editItem` above instead of simply using `() => setSelectedItem(item)` here.
+                    setBlockedActionFunction(() => editItem);
+                  } else {
+                    setSelectedItem(item);
+                  }
+                }}
+              />
+              {errorState?.errors &&
+                errorState.errors.length > 0 &&
+                !selectedItem && (
+                  <Stack gap={1} sx={{ mt: 4 }}>
+                    <ErrorAlert key='errors' errors={errorState.errors} />
+                  </Stack>
+                )}
+            </Box>
+            {dirty && (
+              <Paper sx={{ p: 4 }}>
+                <Stack
+                  direction='row'
+                  justifyContent='space-between'
+                  sx={{ alignItems: 'center' }}
+                >
+                  <Stack direction='row' gap={2}>
+                    <LoadingButton
+                      variant='outlined'
+                      loading={saveLoading}
+                      onClick={() => onSave(workingDefinition)}
+                    >
+                      Save Draft
+                    </LoadingButton>
+                    <Button>Publish</Button>
+                  </Stack>
+                  <Typography variant='body2'>
+                    Last saved on {lastUpdatedDate} by {lastUpdatedBy}
+                  </Typography>
+                </Stack>
+              </Paper>
+            )}
+          </Box>
+        </Box>
         <FormBuilderPalette
           onItemClick={(itemType) => {
             const newItem: FormItem = {
-              linkId: crypto.randomUUID(), // TODO(#6083) - this is a placeholder, but right now will only work in localhost
-              text: itemType.toString(),
+              linkId: crypto.randomUUID(), // TODO(#6083) - this is a placeholder
+              text: FORM_ITEM_PALETTE[itemType].displayName,
```

Comment:
I think this looks a little confusing. In the preview it's almost like "oh, is that just an extra label they put in to tell me what kind of question this is"? Idk. It might be best to just leave this blank, or put something generic like "label" or "untitled question"